icons/meta.ts

```diff
@@ -1778,4 +1778,4 @@ export default {
     'zilliqa-color': 'zilliqa-color',
```

Comment:
This file is auto-generated by `src/scripts/update-icons.js` so we'll need to make this adjustment there too.